src/streamlink/plugins/ard_mediathek.py

```diff
@@ -15,8 +18,8 @@
     0: "144p"
 }
 
-_url_re = re.compile(r"http(s)?://(?:(\w+\.)?ardmediathek.de/tv|mediathek.daserste.de/)")
-_media_id_re = re.compile(r"/play/(?:media|config)/(\d+)")
+_url_re = re.compile(r"http(s)?://(?:(\w+\.)?ardmediathek.de/|mediathek.daserste.de/)")
```

Comment:
`mediathek.daserste.de` can be removed

Reply:
Is this solution not better?:
`_url_re = re.compile(r"http(s)?://(?:(\w+\.)?ardmediathek.de/((?!daserste).)*/|mediathek.daserste.de/)")`

Reply:
I don't think the domain is needed, it just redirects to the new domain.

Reply:
This Link can't be resolved with this plugin, this is better done with **ard_live** or if we rename it to **daserste** .
https://www.ardmediathek.de/daserste/sendung/babylon-berlin/Y3JpZDovL2Rhc2Vyc3RlLmRlL2JhYnlsb24tYmVybGlu/

Reply:
Sorry, I've test it with a wrong link.

Reply:
> mediathek.daserste.de can be removed

It fails with: [http://mediathek.daserste.de/live](http://mediathek.daserste.de/live)